### PR TITLE
Renaming onboarding guide, adding credential guide

### PR DESCRIPTION
**Pull request by mclacore**
This is a big guide. Lots of stuff going on. I had a lot of fun playing around w/ React to get this to work. Might be overkill, might not be. But I think it's cool.

### CODE REVIEW

File: docs/getting_started/02-credentials.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
+---
+id: getting-started-credentials
+slug: /getting-started/credentials
+title: Credentials
+sidebar_label: Credentials
+---
+
+import UUIDProvider, { UUIDContext } from '@site/src/components/UUIDFetcher';
```
Comment (chrisghill) [inline docs/getting_started/02-credentials.md:8]: Whats the javascript stuff you're adding here?

**Review by chrisghill — APPROVED** [review]
Is this mostly copy-pasta'd from the website?

File: src/components/DynamicCodeBlock.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+};
+
+export {
+  DynamicTrustPolicy,
+  DynamicRolePrivileges,
+  DynamicServicePrincipal,
+  DynamicServiceAccount,
+  DynamicProjectPolicy,
+  DynamicAccountKeys,
+};
```
Comment (wasinsandiego) [inline src/components/DynamicCodeBlock.js:86]: You should be able to do the named exports for these where you declare them. Remove this exported object and instead add `export` before each `const` above. It depends on how the es6 modules are transpired, 
 but if it works, this is simpler to maintain.

**Review by wasinsandiego — APPROVED** [review]
This is rad! I dropped a suggestion in slack to make reusable components for the input and display the input's value. Let me know if you have any questions or comments.

File: docs/getting_started/01-prerequisites.md [hunk 3]
```diff
@@ -1,62 +1,57 @@
 ---
-id: getting-started-onboarding
-slug: /getting-started/onboarding
-title: Onboarding
-sidebar_label: Onboarding
+id: getting-started-prerequisites
+slug: /getting-started/prerequisites
```
Comment (coryodaniel) [inline docs/getting_started/01-prerequisites.md:3]: Looks cool! Be sure to add a redirect from /getting-started/onboarding to /getting-started/prerequisites so google seo console does bzzt me.

File: src/components/DynamicCodeBlock.js (hunk 2 above)
Comment (mclacore) [reply src/components/DynamicCodeBlock.js]: Thanks! Added this in!

File: docs/getting_started/01-prerequisites.md (hunk 3 above)
Comment (mclacore) [reply docs/getting_started/01-prerequisites.md:3]: Added, and tested redirect

**Review by xpositivityx — APPROVED** [review]

**Pull request merged**